docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -131,6 +131,19 @@ zeebe:
     runAsUser: 0
 ```
 
+### v8.2.9
+
+#### Optimize
+
+In Optimize 3.10.1, a new environment variable introduced for redirection URL. However, the change was not compatible with Camunda Platform Helm chart until it is fixed in 3.10.3 (and Helm chart 8.2.9). For that reason, those versions are coupled to certain Camunda Platform Helm chart versions.
```

Comment:
A comment for my own understanding, but can you define redirection URL for me?

Reply:
With the users logging into one of the applications, it sends them to the Identity service (namely, Keycloak).
Once the users logged in, they are redirected back to the application. For security, certain redirection URL are allowed (so no one can manipulate them and redirects the users to non-trusted pages).